transform/snowflake-dbt/models/hightouch/freemium/professional_subscription/freemium_account_update.sql

```diff
@@ -8,14 +8,78 @@
 WITH customers_with_cloud_paid_subs as (
     SELECT
         customers_with_cloud_paid_subs.*,
-        'Customer' as account_type
+        'Customer' as account_type,
+        -- Metadata to be used for selecting proper external id
+        external_id_account.id as dwh_id_account_id,
+        external_id_account.id is not null as has_external_id_match,
```

Comment:
why not `has_dwh_id_account_match` instead, to keep in sync with the previous renaming?

Reply:
Replaced.